seed_db.sh

```diff
@@ -0,0 +1,136 @@
+    echo "Warning: \$USER_AUTH_ID is empty. Please set the value to the one posted in the secrets channel." 
+fi 
+
+if [ -z "$ADMIN_AUTH_ID" ] 
+then 
+    echo "Warning: \$ADMIN_AUTH_ID is empty. Please set the value to the one posted in the secrets channel." 
+fi 
+
+if [[ "$1" = "erase" ]]
+then 
+    echo "Erasing the database..."
+
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "DELETE FROM story_translations;"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "DELETE FROM users;"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "DELETE FROM stories;"
+
+    echo "Erased the database 💥💥💥"
+
+elif [[ "$1" = "kevin" ]]
+then 
+    echo "Generating data exposing the corruption, scandal, and graft of Kevin..."
+
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "ALTER SEQUENCE users_id_seq RESTART WITH 1;"
+
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO users (first_name, last_name, auth_id, role, approved_languages) VALUES ('Carl', 'Sagan', '$USER_AUTH_ID', 'User', '{\"ENGLISH_US\":5}');"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO users (first_name, last_name, auth_id, role, approved_languages) VALUES ('Miroslav', 'Klose', '$USER_AUTH_ID', 'User', '{\"POLISH\":5, \"GERMAN\":3}');"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO users (first_name, last_name, auth_id, role, approved_languages) VALUES ('Kevin', 'De Bryune', '$USER_AUTH_ID', 'User', '{\"DUTCH\":5, \"FRENCH\":6}');"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO users (first_name, last_name, auth_id, role, approved_languages) VALUES ('Dwight', 'D. Eisenhower', '$USER_AUTH_ID', 'User', '{\"ENGLISH\":10}');"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO users (first_name, last_name, auth_id, role, approved_languages) VALUES ('Alexander', 'Hamilton', '$USER_AUTH_ID', 'User', '{\"ENGLISH\":3}');"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO users (first_name, last_name, auth_id, role, approved_languages) VALUES ('Angela', 'Merkel', '$ADMIN_AUTH_ID', 'Admin', '{\"GERMAN\":10}');"
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO users (first_name, last_name, auth_id, role, approved_languages) VALUES ('Richard', 'Feynman', '$USER_AUTH_ID', 'User', '{\"PORTUGESE\":3}');"
+
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "ALTER SEQUENCE stories_id_seq RESTART WITH 1;"
+    
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO stories (title, description, youtube_link, level, translated_languages) VALUES ('Kevin Burns Coal to Create Smog', 'He wants to test out the HEPA filter in the new Tesla he got for his birthday', 'link', 4, '{\"GERMAN\", \"ENGLISH_UK\"}');"
```

Comment:
Can you use real youtube links? That way the story cards load real youtube iframes. Feel free to use random youtube videos.